packages/contracts/src/dollar/ERC1155SetUri/ERC1155BurnableSetUri.sol

```diff
@@ -12,11 +12,7 @@ import "./ERC1155SetUri.sol";
  * _Available since v3.1._
  */
 abstract contract ERC1155BurnableSetUri is ERC1155SetUri {
-    function burn(
-        address account,
-        uint256 id,
-        uint256 value
-    ) public virtual {
```

Comment:
I'm concerned that I didn't correctly invoke the solidity custom config settings by running the command mentioned in my comment of this PR. 

Perhaps it has different behavior during `lint-staged`?

Reply:
openzeppelin breaks down functions if it has more than two arguments, for instance:

> ![image](https://user-images.githubusercontent.com/109317559/213389400-373fe396-850a-4ce8-9e72-d7bee25325f2.png)
> two args

> ![image](https://user-images.githubusercontent.com/109317559/213389516-2b20e44a-01d0-457c-bf6a-2508af1161bc.png)
> three args

Reply:
Just curious though if you think I ran the formatters correctly?

Reply:
it looks correct!!